comp.go

```diff
@@ -91,3 +93,11 @@ func equals(a, b any) bool {
 
 	return toString(a) == toString(b)
 }
+
+func isTruthy(v any) bool {
+	return toNumberForLess(v) == 1
```

Comment:
What happens when `v` is `float(2)`?

Reply:
ah yeah looks like it should be truthy too
![image](https://github.com/user-attachments/assets/62e82ded-ec22-4e95-9ace-ea40450a7eb5)

but I guess that's only for the "if" operator, not for == comparison against a bool :( 

![image](https://github.com/user-attachments/assets/9af03d88-0842-4167-b126-d52b6481b721)

![image](https://github.com/user-attachments/assets/ada6c16a-a4bd-4822-bda4-5daab93aee4f)

so maybe I should rename this function to boolEquivalent or something to not confuse folks?  But I think it's behaving as it should (at least how the js library does).

Reply:
nah, `isTruthy` and `isFalsey` is okay, they're good names. I'm having issues with `toNumberForLess` but that's another discussion.